test/integration-test.py

```diff
@@ -265,6 +265,28 @@ def expect(target_time, num):
     expect(now, 0)
     expect(after_grace_period, 1)
 
+def run_certificates_per_name_test():
+    try:
+        handle = subprocess.Popen(
+            '''node test.js --email %s --domains %s --challType %s''' %
+            ('test@ratelimit.me', 'lim.it', 'http-01'),
+            shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        handle.wait()
+        out, err = handle.communicate()
```

Comment:
That's a good point, I hadn't thought of that. This approach seems fine then, but add a comment about why.